Resources/Prototypes/_ES/GameRules/station_events.yml

```
@@ -40,7 +43,37 @@
     startAnnouncement: station-event-gas-leak-start-announcement
     endAnnouncement: station-event-gas-leak-end-announcement
   - type: ESGasLeakRule
+  - type: ESSynchronizedVoteManager
+    votes:
+    - ESVoteGasLeakGases
+    - ESVoteGasLeakLocation
+
+- type: entity
+  id: ESVoteGasLeakGases
+  name: Which Gas Will Leak?
```

```suggestion
  name: Which gas should leak?
```

are these supposed to be title cased. idk what the thought is there